Tools/CISupport/ews-build/steps.py

```diff
@@ -2040,27 +2057,40 @@ def analyzeResults(self):
             compile_without_patch_step = CompileJSCWithoutPatch.name
         compile_without_patch_result = self.getStepResult(compile_without_patch_step)
 
+        patch_id = self.getProperty('patch_id', '')
+        pr_number = self.getProperty('github.number')
+
         if compile_without_patch_result == FAILURE:
-            message = 'Unable to build WebKit without patch, retrying build'
+            if pr_number:
+                message = 'Unable to build WebKit without pull request'
```

Comment:
Should provide some suggestion to the user like below, otherwise users would be confused when they read this.

`Unable to build WebKit without PR, please rebase the PR on ToT`
or
`Unable to build WebKit without PR, please run git-webkit pr again to rebase the PR on ToT`